components/style/keyframes.rs

```diff
@@ -150,11 +153,13 @@ pub struct KeyframesAnimation {
 ///
 /// In practice, browsers seem to try to do their best job at it, so we might
 /// want to go through all the actual keyframes and deduplicate properties.
+#[allow(unsafe_code)]
 fn get_animated_properties(keyframe: &Keyframe) -> Vec<TransitionProperty> {
```

Comment:
This is only called from `add_stylesheet`, which is called from `Stylist::update`, so this is safe.